prac_05/word_occurrences.py

```diff
@@ -1,6 +1,5 @@
 """Program to take string input and count occurrences of each word
```

Comment:
I can only comment on your docstring but you could use:
 `frequency = word_to_count.get(word, 0)
 unique_words[word] = frequency + 1` in your for loop to avoid using an if else.